apps/sequencer/src/providers/eth_send_utils.rs

```diff
@@ -127,14 +127,14 @@ pub async fn get_serialized_updates_for_network(
 
     // Don’t post to Smart Contract if we have 0 updates
     if updates.updates.is_empty() {
-        return Ok("".to_string());
+        return Ok(Vec::new());
     }
 
     let contract_version = provider
         .get_contract(PRICE_FEED_CONTRACT_NAME)
         .ok_or(eyre!("{PRICE_FEED_CONTRACT_NAME} contract is not set!"))?
         .contract_version;
-    drop(provider);
+    // drop(provider);
```

Comment:
Do we need to keep the lock on provider? If we do, we should remove the debug log that states we are releasing it.